"Aborted due to warnings" while packaging

I'm getting the following error when trying to package the app:

```
Running "ts:app" (ts) task
Compiling...
Using tsc v2.3.4
app/components/associations.component.ts(23,6): error TS2307: Cannot find module 'spaces-ng/dist/main'.
app/components/owners.component.ts(9,8): error TS2307: Cannot find module 'spaces-ng/dist/main'.
app/components/owners.component.ts(10,33): error TS2307: Cannot find module 'threatconnect-ng/dist/main'.
app/components/resource.component.ts(9,8): error TS2307: Cannot find module 'spaces-ng/dist/main'.
app/components/resource.component.ts(14,6): error TS2307: Cannot find module 'threatconnect-ng/dist/main'.
app/components/sidebar.component.ts(14,6): error TS2307: Cannot find module 'spaces-ng/dist/main'.
app/main.component.ts(40,6): error TS2307: Cannot find module 'spaces-ng/dist/main'.
app/services/settings.service.ts(5,8): error TS2307: Cannot find module 'spaces-ng/dist/main'.
app/services/settings.service.ts(6,37): error TS2307: Cannot find module 'threatconnect-ng/dist/main'.
app/app.component.ts(14,6): error TS2307: Cannot find module 'spaces-ng/dist/main'.
app/app.module.ts(52,6): error TS2307: Cannot find module 'spaces-ng/dist/main'.
app/app.module.ts(62,8): error TS2307: Cannot find module 'threatconnect-ng/dist/main'.
app/shared/indicator.service.ts(15,6): error TS2307: Cannot find module 'spaces-ng/dist/main'.
app/shared/indicator.service.ts(20,6): error TS2307: Cannot find module 'threatconnect-ng/dist/main'.

>> 14 non-emit-preventing type warnings  
>> Error: tsc return code: 2
Warning: Task "ts:app" failed. Use --force to continue.

Aborted due to warnings.
```

I'm running linux on a vagrant box (https://atlas.hashicorp.com/ubuntu/boxes/trusty64) and am trying to package it using:
```
npm install
grunt build:dist
```

--- a/app/app.component.ts
+++ b/app/app.component.ts
@@ -11,7 +11,7 @@ import {
     SpacesMessagesService,
     SpacesStorageService
 }
-from 'spaces-ng/dist/main';
+from 'spaces-ng/';
 
 @Component({
     templateUrl: 'app/app.component.html',

--- a/app/app.module.ts
+++ b/app/app.module.ts
@@ -49,7 +49,7 @@ import {
     SpacesStorageService,
     SpacesUtilityService
 }
-from 'spaces-ng/dist/main';
+from 'spaces-ng/';
 import {
     // Address,
     // EmailAddress,
@@ -59,7 +59,7 @@ import {
     TcIndicatorService,
     TcOwnerService
     // Url
-} from 'threatconnect-ng/dist/main';
+} from 'threatconnect-ng/';
 import { SettingsService } from './services/settings.service';
 
 @NgModule({

--- a/app/components/associations.component.ts
+++ b/app/components/associations.component.ts
@@ -20,13 +20,13 @@ import {
     SpacesBaseService,
     SpacesLoggingService
 }
-from 'spaces-ng/dist/main';
+from 'spaces-ng/';
 
 // import {
 //     TcGroupService,
 //     TcIndicatorService,
 // }
-// from 'threatconnect-ng/dist/main';
+// from 'threatconnect-ng/';
 
 @Component({
     templateUrl: './app/components/associations.component.html',

--- a/app/components/owners.component.ts
+++ b/app/components/owners.component.ts
@@ -6,8 +6,8 @@ from '@angular/core';
 import {
     SpacesLoggingService,
     SpacesMessagesService
-} from 'spaces-ng/dist/main';
-import { TcOwnerService, } from 'threatconnect-ng/dist/main';
+} from 'spaces-ng/';
+import { TcOwnerService, } from 'threatconnect-ng/';
 
 @Component({
     templateUrl: './app/components/owners.component.html',

--- a/app/components/resource.component.ts
+++ b/app/components/resource.component.ts
@@ -6,12 +6,12 @@ from '@angular/core';
 import {
     SpacesLoggingService,
     SpacesMessagesService
-} from 'spaces-ng/dist/main';
+} from 'spaces-ng/';
 import {
     TcGroupService,
     TcIndicatorService,
 }
-from 'threatconnect-ng/dist/main';
+from 'threatconnect-ng/';
 
 @Component({
     templateUrl: './app/components/resource.component.html',

--- a/app/components/sidebar.component.ts
+++ b/app/components/sidebar.component.ts
@@ -11,7 +11,7 @@ import {
     SpacesMessagesService,
     SpacesStorageService
 }
-from 'spaces-ng/dist/main';
+from 'spaces-ng/';
 
 @Component({
     templateUrl: './app/components/sidebar.component.html',

--- a/app/main.component.ts
+++ b/app/main.component.ts
@@ -37,7 +37,7 @@ import {
     SpacesMessagesService,
     SpacesStorageService
 }
-from 'spaces-ng/dist/main';
+from 'spaces-ng/';
 
 /*
 import {
@@ -46,7 +46,7 @@ import {
     TcIndicatorService,
     TcOwnerService
 }
-from 'threatconnect-ng/dist/main';
+from 'threatconnect-ng/';
 */
 
 import { OwnersComponent } from './components/owners.component';

--- a/app/services/settings.service.ts
+++ b/app/services/settings.service.ts
@@ -2,8 +2,8 @@ import { Injectable } from '@angular/core';
 import {
     SpacesLoggingService,
     SpacesMessagesService
-} from 'spaces-ng/dist/main';
-import { TcExchangeDbService } from 'threatconnect-ng/dist/main';
+} from 'spaces-ng/';
+import { TcExchangeDbService } from 'threatconnect-ng/';
 
 
 @Injectable()

--- a/app/shared/indicator.service.ts
+++ b/app/shared/indicator.service.ts
@@ -12,12 +12,12 @@ from 'primeng/primeng';
 import {
     SpacesLoggingService
 }
-from 'spaces-ng/dist/main';
+from 'spaces-ng/';
 
 import {
     TcIndicatorService
 }
-from 'threatconnect-ng/dist/main';
+from 'threatconnect-ng/';
 
 
 @Injectable()

--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
     "ngx-bowser": "^0.1.0",
     "primeng": "2.0.0",
     "primeui": "^4.1.15",
-    "threatconnect-ng": "git+https://github.com/ThreatConnect-Inc/threatconnect-ng.git#develop"
+    "threatconnect-ng": "git+https://github.com/ThreatConnect-Inc/threatconnect-ng.git"
   },
   "devDependencies": {
     "@angular/common": "2.4.5",

